vue/src/components/leaflet/LeafletComp.vue

```diff
@@ -36,16 +36,39 @@ defineExpose({
 
 onMounted(() => {
   leafletMap = L.map("map", {
-    zoomControl: props.zoomControl,
+    zoomControl: false,
     attributionControl: false,
     zoomDelta: 0.25,
     zoomSnap: 0.25,
   });
 
-  L.tileLayer("https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png", {
-    maxNativeZoom: 18,
-    maxZoom: props.maxZoom,
-  }).addTo(leafletMap);
+  //https://stackoverflow.com/a/55767702
+  if (props.zoomControl) {
+    L.control
+      .zoom({
+        position: "bottomright",
```

Comment:
shouldn't we rather use **topright**?

![grafik](https://user-images.githubusercontent.com/87453514/176043336-d0637caa-76b6-42d2-96e2-8fed5a599600.png)

Reply:
You are absolutely right. 👍🏻